doc/version_1/040_treesForVulMgmt.md

```diff
@@ -192,7 +191,7 @@ Table 7: Utility to the Adversary, as a Combination of Virulence and Value Densi
 | **rapid** | **concentrated** | super effective |
 
 
-Alternative heuristics for proxying adversary utility are plausible. One such example is the value the vulnerability would have were it sold on the open market. Some firms, such as [Zerodium](https://zerodium.com/program.html), make such pricing structures public. The valuable exploits track the virulence and value density heuristics for the most part. Within a single system—whether it is Apache, Windows, iOS or WhatsApp—more automated kill chain steps successfully leads to higher exploit value. Remote code execution with sandbox escape and without user interaction are the most valuable exploits, and those features describe automation of the relevant kill chain steps. How equivalently virulent exploits for different systems are priced relative to each other is more idiosyncratic. Price does not only track value density of the system, but presumably also the existing supply of exploits and the installation distribution among the targets of Zerodium’s customers. Currently, we simplify the analysis and ignore these factors. However, future work should look for and prevent large mismatches between the outputs of the *utility* decision point and the exploit markets.
+Alternative heuristics for proxying adversary utility are credible. One such example is the value the vulnerability would have were it sold on the open market. Some firms, such as [Zerodium](https://zerodium.com/program.html), make such pricing structures public. The valuable exploits track the virulence and value density heuristics for the most part. Within a single system—whether it is Apache, Windows, iOS or WhatsApp—more automated kill chain steps successfully leads to higher exploit value. Remote code execution with sandbox escape and without user interaction are the most valuable exploits, and those features describe automation of the relevant kill chain steps. How equivalently virulent exploits for different systems are priced relative to each other is more idiosyncratic. Price does not only track value density of the system, but presumably also the existing supply of exploits and the installation distribution among the targets of Zerodium’s customers. Currently, we simplify the analysis and ignore these factors. However, future work should look for and prevent large mismatches between the outputs of the *utility* decision point and the exploit markets.
```

Comment:
My memory is that this line is not using "plausible" in the sense of reasoning about a worst-plausible result of an exploit. "Plausible" here is just introducing that the authors might make some other choices in the future about how the decision point is evaluated. So I think keeping it as "plausible" here would make this difference clearer, there was no way to tell in the previous text that this use of "plausible" was different than the earlier ones.

Reply:
I agree.  That was one substitution I suspected was unintended.  
Thank you